client/src/app/components/TargetCard.tsx

```diff
@@ -68,6 +69,8 @@ export const TargetCard: React.FC<TargetCardProps> = ({
   const { t } = useTranslation();
   const [isCardSelected, setCardSelected] = React.useState(cardSelected);
 
+  const [imageDataUrl, setImageDataUrl] = React.useState<string | null>(null);
```

Comment:
It's not really a url anymore, it's just the image data itself, so.... s/imageDataUrl/imageData/

Reply:
Ok, so reading it again today (with less blurry eyes and more coffee) and it really is a data-url.  So the name does make sense!

Reply:
+1 Updated